JavaScript/JavaScriptSDK/Context/Session.ts

```diff
@@ -76,11 +79,14 @@ module Microsoft.ApplicationInsights.Context {
             // renew if acquisitionSpan or renewalSpan has ellapsed
             if (acquisitionExpired || renewalExpired) {
                 // update automaticSession so session state has correct id                
-                this.automaticSession.isFirst = undefined; 
+                this.automaticSession.isFirst = undefined;
                 this.renew();
```

Comment:
Thanks for spotting this. Currently `renew()` is only called from `update()` (which has a throttling check) and `initializeAutomaticSession()` (which is only called once per page view). I agree that a throttled function is a right solution, but since it is not currently required I stick with the current implementation.